src/diagnostic.c

```diff
@@ -51,210 +51,210 @@
-    [PM_ERR_TERNARY_EXPRESSION_TRUE]                  = "Expected an expression after `?` in the ternary operator",
-    [PM_ERR_UNDEF_ARGUMENT]                           = "Invalid argument being passed to `undef`; expected a bare word, constant, or symbol argument",
-    [PM_ERR_UNARY_RECEIVER_BANG]                      = "Expected a receiver for unary `!`",
-    [PM_ERR_UNARY_RECEIVER_MINUS]                     = "Expected a receiver for unary `-`",
-    [PM_ERR_UNARY_RECEIVER_PLUS]                      = "Expected a receiver for unary `+`",
-    [PM_ERR_UNARY_RECEIVER_TILDE]                     = "Expected a receiver for unary `~`",
-    [PM_ERR_UNTIL_TERM]                               = "Expected an `end` to close the `until` statement",
-    [PM_ERR_VOID_EXPRESSION]                          = "Unexpected void value expression",
-    [PM_ERR_WHILE_TERM]                               = "Expected an `end` to close the `while` statement",
-    [PM_ERR_WRITE_TARGET_READONLY]                    = "Immutable variable as a write target",
-    [PM_ERR_WRITE_TARGET_UNEXPECTED]                  = "Unexpected write target",
-    [PM_ERR_XSTRING_TERM]                             = "Expected a closing delimiter for the `%x` or backtick string",
-    [PM_WARN_AMBIGUOUS_FIRST_ARGUMENT_MINUS]          = "Ambiguous first argument; put parentheses or a space even after `-` operator",
-    [PM_WARN_AMBIGUOUS_FIRST_ARGUMENT_PLUS]           = "Ambiguous first argument; put parentheses or a space even after `+` operator",
-    [PM_WARN_AMBIGUOUS_PREFIX_STAR]                   = "Ambiguous `*` has been interpreted as an argument prefix",
-    [PM_WARN_AMBIGUOUS_SLASH]                         = "Ambiguous `/`; wrap regexp in parentheses or add a space after `/` operator",
+    [PM_ERR_ALIAS_ARGUMENT]                     = "Invalid argument being passed to `alias`; expected a bare word, symbol, constant, or global variable",
+    [PM_ERR_AMPAMPEQ_MULTI_ASSIGN]              = "Unexpected `&&=` in a multiple assignment",
+    [PM_ERR_ARGUMENT_AFTER_BLOCK]               = "Unexpected argument after a block argument",
+    [PM_ERR_ARGUMENT_AFTER_FORWARDING_ELLIPSES] = "Unexpected argument after `...`",
+    [PM_ERR_ARGUMENT_BARE_HASH]                 = "Unexpected bare hash argument",
+    [PM_ERR_ARGUMENT_BLOCK_MULTI]               = "Multiple block arguments; only one block is allowed",
+    [PM_ERR_ARGUMENT_FORMAL_CLASS]              = "Invalid formal argument; formal argument cannot be a class variable",
+    [PM_ERR_ARGUMENT_FORMAL_CONSTANT]           = "Invalid formal argument; formal argument cannot be a constant",
+    [PM_ERR_ARGUMENT_FORMAL_GLOBAL]             = "Invalid formal argument; formal argument cannot be a global variable",
+    [PM_ERR_ARGUMENT_FORMAL_IVAR]               = "Invalid formal argument; formal argument cannot be an instance variable",
+    [PM_ERR_ARGUMENT_FORWARDING_UNBOUND]        = "Unexpected `...` in an non-parenthesized call",
```

Comment:
@kddnewton Does this error seem good to you?

Reply:
Looks good!